src/components/passphrase/steps.js

```diff
@@ -0,0 +1,45 @@
+export default context => ({
+  info: {
+    cancelButton: {
+      title: 'cancel',
+      onClick: () => { context.props.closeDialog(); },
+    },
+    confirmButton: {
+      title: 'next',
+      onClick: () => { context.setState({ currentStep: 'generate' }); },
+    },
+  },
+  generate: {
+    cancelButton: {
+      title: 'cancel',
+      onClick: () => { context.props.closeDialog(); },
+    },
+    confirmButton: {
+      title: 'Yes! It\'s safe',
+      onClick: () => {},
+    },
+  },
+  show: {
```

Comment:
I just realized that the `show` step is missing the title "Save your passphrase in a safe place!".  Without the title the "yes it's safe" button doesn't make sense